src/MudBlazor/MudBlazor.csproj

```
@@ -58,13 +58,13 @@
   </Target>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-    <PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.16" />
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.25" />
     <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.16" />
     <PackageReference Include="Microsoft.Extensions.Localization" Version="6.0.16" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
-    <PackageReference Include="Microsoft.AspNetCore.Components" Version="7.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.25" />
```

Should it be 7.0.5 for net7?

```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Components" Version="7.0.5" />
```

---

No, it should be 7.0.14, but dependabot doesn't work well with multitargets.